src/options/smt_options.toml

```diff
@@ -412,13 +412,24 @@ header = "options/smt_options.h"
   help       = "in models, output arrays (and in future, maybe others) using abstract values, as required by the SMT-LIB standard"
 
 [[option]]
-  name       = "modelUninterpDtEnum"
-  category   = "regular"
-  long       = "model-u-dt-enum"
-  type       = "bool"
-  default    = "false"
-  read_only  = true
-  help       = "in models, output uninterpreted sorts as datatype enumerations"
+  name       = "modelUninterpPrint"
+  smt_name   = "model-uninterp-print"
+  category   = "regular"
+  long       = "model-u-print=MODE"
+  type       = "ModelUninterpPrintMode"
+  default    = "DeclFun"
+  read_only  = true
+  help       = "determines how to print uninterpreted elements in models"
+  help_mode  = "uninterpreted elements in models printing  modes."
+[[option.mode.DtEnum]]
+  name = "dtenum"
+  help = "print uninterpreted elements as datatype enumerations, where the sort is the datatype"
+[[option.mode.DeclSortAndFun]]
+  name = "decl-sort-and-fun"
+  help = "print uninterpreted elements declare-fun, and also include a declare-sort for the sort"
+[[option.mode.DeclFun]]
```

Comment:
My understanding the SMT-LIB standard was that we wouldn't include declare-fun either, is this right?  In this case, I'd expect another option, which would be "None" for don't print anything.

Reply:
I agree, but my plan was to hold off with the "none" option until we resolve the `as` issue. Printing neither `declare-fun` nor `as` is not very intuitive in my opinion.

Reply:
Ah ok, thats fine.  Then the default will change to None afterwards?

Reply:
yes